### ISSUE TITLE
Can't install anything, problem with PGP signatures

### ISSUE THREAD

**Issue by vincent456** [yuk7/ArchWSL#266]
- I've done fresh install.
-  Then when I `pacman -Syu` I got the following error message `warning: Public keyring not found; have you run 'pacman-key --init'?
downloading required keys...
error: keyring is not writable`
- Following the message, I run `pacman-key --init`
- Then again `pacman -Syu`  it keeps asking me to import PGP keys. That's suspicious but I did it. Then I got the error `error: archlinux-keyring: signature from "Christian Hesse (Arch Linux Package Signing) <arch@eworm.de>" is unknown trust:: File /var/cache/pacman/pkg/archlinux-keyring-20211028-1-any.pkg.tar.zst is corrupted (invalid or corrupted package (PGP signature)).
Do you want to delete it? [Y/n]`
- I also tried just installing archlinux-keyring but it also failed.
- In conclusion, I can't install anything.

**Comment by yuk7** [comment]
Hi, Thank you for reporting.
Did you run populate?
```
sudo pacman-key --populate
```

https://wsldl-pg.github.io/ArchW-docs/How-to-Setup/

**Comment by vincent456** [comment]
It worked ! thank you ! I didn't noticed the link to the setup page.
